tests/e2e/tests/AggregatesSection/AggregatesShouldNotBeZeroInTriggerState.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, test } from '@playwright/test';
+import AggregatesComponent from 'Pages/AggregatesComponent';
+import DashboardPage from 'Pages/DashboardPage';
+import UserStateComponent from 'Pages/UserStateComponent';
+import { qase } from 'playwright-qase-reporter';
+import { TriggerDataSet } from 'testData/testData.enum';
+
+import {
+  getAccessToken,
+  mockFloods,
+  resetDB,
+} from '../../helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Login
+  const loginPage = new LoginPage(page);
+
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+  // We should maybe create one mock for all different disaster types for now we can just use floods
+  await mockFloods(
+    TriggerDataSet.TriggerScenario,
+    TriggerDataSet.CountryCode,
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(TriggerDataSet.UserMail, TriggerDataSet.UserPassword);
+});
+
+test(
+  qase(39, '[Trigger] Aggregates values should be non-zero'),
+  async ({ page }) => {
+    const aggregates = new AggregatesComponent(page);
+    const dashboard = new DashboardPage(page);
+    const userState = new UserStateComponent(page);
+
+    // Navigate to disaster type the data was mocked for
+    await dashboard.navigateToFloodDisasterType();
+    // Assertions
+    await userState.headerComponentIsVisible({
+      countryName: TriggerDataSet.CountryName,
+    });
+
+    // get the number of warning events and aggregated events
+    const aggregatesEventCount = await aggregates.getNumberOfPredictedEvents();
+
+    // check if the number of warning events is equal to the number of aggregated events
+    expect(aggregatesEventCount).toBeGreaterThan(0);
```

Comment:
Given that you only test th number of events, I would reformulate the test description to be more specific (instead of currently "aggregate values".

Reply:
I changed it to: [Trigger] Aggregated number of events should be non-zero